docs/building-apps/moneygram-access-integration-guide.mdx

```diff
@@ -142,6 +157,22 @@ The following code demonstrates how to implement the application’s side of thi
 
 <CodeExample>
 
+```ts
+import { Wallet, SigningKeypair } from "@stellar/typescript-wallet-sdk";
+
+const wallet = Wallet.TestNet();
+
+// First we create an anchor object with MoneyGram's home domain.
+const anchor = wallet.anchor({ homeDomain: MGI_ACCESS_HOST });
```

Comment:
Can you set this variable in the code please?
Maybe something like
```typescript
// Testnet
// MGI_ACCESS_HOST = "extstellar.moneygram.com"
// Main net
MGI_ACCESS_HOST = "stellar.moneygram.com"
```

Reply:
Sure thing. I was using Python's examples/vars as reference.

I'll set the var to make it more clear.